test_tools_list.py

```
@@ -0,0 +1,94 @@
+    full_message = f"Content-Length: {content_length}\r\n\r\n{json_content}"
+    
+    print(f"Testing tools/list with message:")
+    print(f"Content-Length: {content_length}")
+    print(f"JSON: {json_content}")
+    print()
+    
+    try:
+        # Start the MCP server
+        process = subprocess.Popen(
+            ["/Users/derekvitrano/Developer/Work/tools/Graph-of-Thought-MCP/venv/bin/python",
+             "/Users/derekvitrano/Developer/Work/tools/Graph-of-Thought-MCP/mcp-wrapper/mcp_server.py"],
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)